civictechprojects/views.py

```diff
@@ -344,19 +344,13 @@ def index(request, id='Unused but needed for routing purposes; do not remove!'):
     if redirect_result is not None:
         return redirect(redirect_result)
 
-    # TODO: Add to redirectors.py
-    page_url = request.get_full_path()
-    clean_url = get_clean_url(page_url)
-    if clean_url != page_url:
-        print('Redirecting unclean {old_url} to {new_url}'.format(old_url=page_url, new_url=clean_url))
-        return redirect(clean_url)
+    redirect_result = redirect_by([InvalidArgumentsRedirector, DirtyUrlsRedirector, DeprecatedUrlsRedirector], request.get_full_path())
+    if redirect_result is not None:
+        return redirect(redirect_result)
 
-    # TODO: Add to redirectors.py
-    section_name = get_page_section(clean_url)
-    deprecated_redirect_url = redirect_from_deprecated_url(section_name)
-    if deprecated_redirect_url:
-        print('Redirecting deprecated url {name}: {url}'.format(name=section_name, url=clean_url))
-        return redirect(deprecated_redirect_url)
+    redirect_result = redirect_by([DeprecatedUrlsRedirector], request.get_full_path())
+    if redirect_result is not None:
+        return redirect(redirect_result)
```

Comment:
This is unneeded, covered by line 347 above.  Same with Line 343.

Reply:
Yeah, I am pushing the changes.